docs/en/Getting-Started.md

```diff
@@ -343,6 +344,33 @@ the following command:
 $ ./bin/alluxio-stop.sh local
 ```
 
+## Diagnose Alluxio startup issue
+
+If you have problems starting Alluxio in your local environment, try the following command:
```

Comment:
Yeah I put the local version of command at first because the whole page is a tutorial that runs Alluxio on local environment. I did mention the cluster version afterwards in case they wanted to validate the cluster environment.